checkpoint.go

```diff
@@ -40,13 +42,14 @@ checkpointed.`,
 		}
 		defer destroy(container)
 		options := criuOptions(context)
+		preDump := context.Bool("pre-dump")
```

Comment:
Why don't we add that as part of `criuOptions`? I don't like that the interface for checkpointing now takes two arguments which are both technically "criu options".